Cake.Console/Cake.Console.csproj

```
@@ -8,6 +8,7 @@
     <PackageReference Include="cake.core" Version="1.2.0" />
     <PackageReference Include="cake.common" Version="1.2.0" />
     <PackageReference Include="cake.Nuget" Version="1.2.0" />
+    <PackageReference Include="cake.cli" Version="1.2.0" />

```

Package id has casing (NuGet normalizes so doesn't really matter)
```suggestion
    <PackageReference Include="Cake.Cli" Version="1.2.0" />

```

Reference example from NuGet
https://www.nuget.org/packages/Cake.CLI